include/tvm/runtime/ndarray.h

```diff
@@ -30,8 +30,11 @@ class NDArray {
    */
   explicit inline NDArray(Container* data);
   /*!
-   * \brief copy constructor
-   * \param other The value to be copied
+   * \brief copy constructor.
+   *
+   * Its not a deep copy but ref count increment instead.
```

Comment:
It does not make a copy, but a reference count of the input NDArray is incremented